### PR TITLE
Cleanup package.json

### PR DESCRIPTION
**Pull request by FDiskas**
package.json upgrade and cleanup

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,17 +43,18 @@
...
+    "butter-component-settings": "^0.2.0",
+    "butter-provider": "github:butterproviders/butter-provider",
+    "butter-provider-archive": "github:butterproviders/butter-provider-archive",
+    "butter-provider-ccc": "github:butterproviders/butter-provider-ccc",
+    "butter-provider-vodo": "github:butterproviders/butter-provider-vodo",
+    "butter-provider-youtube": "github:butterproviders/butter-provider-youtube",
+    "butter-theme-dark": "github:butterthemes/butter-theme-dark",
+    "butter-theme-flax": "github:butterthemes/butter-theme-flax",
+    "butter-theme-pink": "github:butterthemes/butter-theme-pink",
+    "butter-theme-yellow": "github:butterthemes/butter-theme-yellow",
```
Comment (xaiki) [inline package.json]: i think the former is the correct syntax

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "backbone.marionette": "marionettejs/backbone.marionette#^2.4.1",
+    "mousetrap": "ccampbell/mousetrap#~1.6.0",
+    "underscore": "jashkenas/underscore#^1.8.3",
+    "video.js": "videojs/video.js#4.11.4",
+    "videojs-youtube": "eXon/videojs-youtube#1.2.10"
+  },
+  "resolutions": {
+    "backbone": ">=0.9.9 <=1.3.x"
+  }
+}
```
Comment (xaiki) [inline bower.json]: we removed that on purpose.

File: package.json [hunk 3]
```diff
@@ -93,26 +94,14 @@
...
-    "@bower_components/bootstrap": "twbs/bootstrap#~3.3.7",
-    "@bower_components/dosis-fontface": "elartix/dosis-fontface#^0.0.1",
-    "@bower_components/font-awesome": "FortAwesome/Font-Awesome#^4.6.3",
-    "@bower_components/jquery": "jquery/jquery-dist#1.9.1 - 3",
-    "@bower_components/marionette": "marionettejs/backbone.marionette#^2.4.1",
-    "@bower_components/mousetrap": "ccampbell/mousetrap#~1.6.0",
-    "@bower_components/underscore": "jashkenas/underscore#>=1.7.0",
-    "@bower_components/video.js": "videojs/video.js#4.11.4",
-    "@bower_components/videojs-youtube": "eXon/videojs-youtube#1.2.10"
+    "webtorrent-health": "1.x.x"
```
Comment (xaiki) [inline package.json]: dito

File: package.json [hunk 4]
```diff
@@ -93,26 +94,14 @@
...
   },
   "devDependencies": {
     "bower": "^1.7.9",
-    "butter-component-builder": "buttercomponents/butter-component-builder",
```
Comment (xaiki) [inline package.json]: that's needed for working with react components.

File: package.json [hunk 5]
```diff
@@ -127,6 +116,7 @@
     "nw-builder": "^3.1.2",
     "run-sequence": "^1.1.5",
     "stylus": "^0.54.2",
+    "webpack": "^4.5.0",
```
Comment (xaiki) [inline package.json]: that's not needed by itself

Comment (FDiskas) [reply package.json]: Did you tried to boot up this project from scratch?

**Comment by FDiskas** [comment]
You can close this pull request. I just tried to boot up this project and added or fixed dependencies that prevented to run this project correctly :oncoming_taxi:

**Pull request closed without merging**